acorn/src/expression.js

```diff
@@ -455,6 +463,22 @@ pp.parseDynamicImport = function(node) {
   return this.finishNode(node, "ImportExpression")
 }
 
+pp.parseImportMeta = function(node) {
+  this.next() // skip `.`
+
+  const containsEsc = this.containsEsc
+  node.property = this.parseIdent(true)
+
+  if (node.property.name !== "meta")
+    this.raiseRecoverable(node.property.start, "The only valid meta property for import is import.meta")
+  if (containsEsc)
+    this.raiseRecoverable(node.start, "'import.meta' must not contain escaped characters")
```

Comment:
Do we have same error in `new.target`? We probably would want these error messages to align.

Reply:
I have pushed 0164a93 to align the error messages.

For a reference, "'import.meta' must not contain escaped characters" is the same message as Chrome 80.

Reply:
I don't think it matters much to align with some engine, but internal consistency is important. Thanks for fixing!